docs/tutorials/t2hahn_characterization.ipynb

```diff
@@ -0,0 +1,590 @@
+      "text/plain": [
+       "<Figure size 576x360 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from qiskit_experiments.test.t2hahn_backend import T2HahnBackend\n",
+    "\n",
+    "estimated_t2hahn2 = 20 * conversion_factor\n",
+    "# The behavior of the backend is determined by the following parameters\n",
+    "backend2 = T2HahnBackend(\n",
+    "    t2hahn=[estimated_t2hahn2],\n",
+    "    frequency=[200010],\n",
+    "    initialization_error=[0.0],\n",
+    "    readout0to1=[0.02],\n",
+    "    readout1to0=[0.02],)\n",
+    "\n",
+    "\n",
+    "# Analysis for Hahn Echoe experiemnt with 0 echoes.\n",
+    "expdata2_0echoes = exp2_0echoes.run(backend=backend2, shots=2000)\n",
+    "expdata2_0echoes.block_for_results()  # Wait for job/analysis to finish.\n",
+    "\n",
+    "# Analysis for Hahn Echoe experiemnt with 4 echoes.\n",
+    "expdata2_4echoes = exp2_4echoes.run(backend=backend2, shots=2000)\n",
+    "expdata2_4echoes.block_for_results()  # Wait for job/analysis to finish.\n",
+    "\n",
+    "# Display the figure\n",
+    "print(\"Hahn Echoe with 0 echoes:\")\n",
+    "display(expdata2_0echoes.figure(0))\n",
+    "print(\"Hahn Echoe with 4 echoes:\")\n",
+    "display(expdata2_4echoes.figure(0))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As We can see, the estimate $T_2$ is different in the two plots. The mock backend for this experiment used $T_{2} = 20[\\mu s]$ so we can see that the estimation with 4 echoes is better."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "###  $T_{2}$ versus  $T_{2}^{\\ast}$\n",
+    "This experiment purpose is to give a better estimate for the dephasing noise. In Ramsey experiment, we can estimate $T_{2}^{\\ast}$ but this is not truly the dephasing noise as the information is not lost.\n",
+    "The $\\ast$ indicates that $T_{2}^{\\ast}$ is sensitive to inhomogeneous broadening. This affect the qubit frequncy.\n",
+    "In Ramsey experiment, we estimate the frequency of the qubit while in Hahn Echo experiment there is no need.\\\n",
```

Comment:
In Ramsey you estimate the frequency error, not the qubit frequency

Reply:
This section was removed from the tutorial.